infrastructure/lib/constructs/database.ts

```diff
@@ -36,12 +36,24 @@ class DatabaseBootstrapper extends cdk.Construct {
   public readonly backingLambda: lambda.IFunction;
   constructor(scope: cdk.Construct, id: string, props: BootstrapProps) {
     super(scope, id);
+
+    const layer = new lambda.LayerVersion(this, "DatabaseSupportLayer", {
+      compatibleRuntimes: [lambda.Runtime.NODEJS_14_X],
```

Comment:
Is this an alternative to specify an exact image where the bundling will be executed?

Reply:
This just decreases the value from "all runtimes" https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda.LayerVersion.html#compatibleruntimes. It doesn't _really_ matter. But I didn't want to add things we don't know work.